docs/marketplace/user/merchant-portal-user-guides/202200.0/products/abstract-products/managing-marketplace-abstract-product.md

```diff
@@ -0,0 +1,83 @@
+2. In the *Name & Description* pane, edit *Name* and *Description* for every locale.
+3. In the *Stores* pane, in the drop-down menu select the stores where the product is available.
+4. In the *Tax Set* pane, in the drop-down menu, select the necessary tax set.
+5. In the *Categories* pane, in the drop-down menu, select the categories where your product belongs.
+6. Click **Save**.  
+
+
+### Reference information: [Product name] drawer, Abstract Product Details tab
+
+| ATTRIBUTE   | DESCRIPTION  | REQUIRED? |
+| ----------- | -------------- | --------- |
+| Name        | Name of your product displayed on the Storefront. It is set per locale. | ✓         |
+| Description | Description of the product displayed on the Storefront. It is set per locale. |           |
+| Stores      | Defines the [stores](/docs/scos/dev/tutorials-and-howtos/howtos/howto-set-up-multiple-stores.html) the product is available in.<br/>You can select multiple values. |           |
+| Price       | In this pane, you can manage prices for your abstract product. See [Managing abstract product prices](/docs/marketplace/user/merchant-portal-user-guides/{{page.version}}/products/abstract-products/managing-marketplace-abstract-product-prices.html) for more details. |           |
+| Tax Set     | The conditions under which a product is going to be taxed.<br/>The values available for selection derive from Taxes > Tax Sets<br/>Only one value can be selected. | ✓         |
+| Images      | In this pane, you can manage image sets for your abstract product. See [Managing abstract product image sets](/docs/marketplace/user/merchant-portal-user-guides/{{page.version}}/products/abstract-products/managing-marketplace-abstract-product-image-sets.html) for more details. |           |
+| Attributes  | In this pane, you can manage attributes for your product. See [Managing abstract product attributes](/docs/marketplace/user/merchant-portal-user-guides/{{page.version}}/products/abstract-products/managing-marketplace-abstract-product-attributes.html) for more details. |           |
+| Categories  | Defines the [categories](/docs/scos/user/features/{{page.version}}/category-management-feature-overview.html) the product is displayed in. |           |
+| SEO         | In this pane, you can manage meta information for your product. See [Managing abstract product meta information](/docs/marketplace/user/merchant-portal-user-guides/{{page.version}}/products/abstract-products/managing-marketplace-abstract-product-meta-information.html) for more details. |           |
+
+### Sending the product for approval
+
+In order for the product to be available on the Storefront, it needs to be approved. To send the product for approval:
+
+1. Next to the abstract product, you want to edit, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *[Product name]* drawer, *Abstract Product Details* tab.
```

Comment:
we will not have for already existing products, only new, and this doc is missing approval process when creating a new product by merchants in the merchant portal